### PR TITLE
Add golang script to download github teams

### PR DESCRIPTION
**Pull request by so0k**
This PR is to add golang script for reading GitHub teams into TF Config.

It also adds Golang dependency management using `Glide`

Sample output:

```hcl
module "backend" {
  source     = "git@github.com:honestbee/tf-modules.git?ref=master//github/team"
  user_roles = {
    so0k = "maintainer"
    terrych0u = "member"
...
    tuannvm = "maintainer"
  }

  team_id = "123456"
}
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Honestbee Devops Tools
+
+Misc Devops Tools used by Honestbee
+
+## Golang Dependency management
+
+Using glide, get all vendored dependencies with:
+
+```
+glide up
```
Comment (tuannvm) [inline README.md]: 👍

File: github-tf/main.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	"fmt"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+func main() {
+	ctx := context.Background()
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: "a0bb39b234303d2a30fe1f8a0de5218a82286703"},
```
Comment (tuannvm) [inline github-tf/main.go]: Could you make it configurable via env / command-line argument?

Comment (so0k) [reply github-tf/main.go]: well duh

**Comment by so0k** [comment]
Done, it works and I'm not going to make any more changes

**Pull request merged**